docs/src/pages/[platform]/getting-started/migration/migration.react.mdx

```diff
@@ -50,7 +50,7 @@ For customers with applications using right to left directionality, the example
 - <View dir="rtl">
 + <ThemeProvider direction="rtl">
 ```
-Also see: [Internationalization](react/getting-started/internationalization)
```

Comment:
This would go to http://localhost:3000/react/getting-started/react/getting-started/internationalization